packages/app/src/specs/CreateSpecCard.cy.tsx

```diff
@@ -55,11 +54,6 @@ describe('<CreateSpecCard />', { viewportWidth: 400, viewportHeight: 400 }, () =
     </div>))
     .get(specCardSelector)
     .click()
-
     .get('@onClickSpy').should('have.been.calledOnce')
-
-    // keyboard checks that should work through focusing a button inside the card:
-    // .type('{enter}')
-    // .type(' ')
```

Comment:
UNIFY-878 has completed but does not impact this. Regardless, this should be validated with the Card's component tests